RELEASE-NOTES.md

```diff
@@ -12,6 +12,7 @@
 * Added support for fully qualified image repository names in `spcs image-repository` commands.
 * Added `--if-not-exists` option to `create` commands for `service`, and `compute-pool`. Added `--replace` and `--if-not-exists` options for `image-repository create`.
 * Added support for python connector diagnostic report.
+* `snow sql` command supports now templating of queries.
```

Comment:
Can we clarify this to: supports client-side templating of queries?

Reply:
Added